src/common/database.cpp

```diff
@@ -154,13 +154,12 @@ void YamlDatabase::parse( const ryml::Tree& tree ){
 	if( this->nodeExists( tree.rootref(), "Body" ) ){
 		const ryml::NodeRef& bodyNode = tree["Body"];
 		size_t childNodesCount = bodyNode.num_children();
-		size_t childNodesProgressed = 0;
 		const char* fileName = this->currentFile.c_str();
 
+		ShowStatus("Loading '" CL_WHITE "%" PRIdPTR CL_RESET "' entries in '" CL_WHITE "%s" CL_RESET "'" CL_CLL "\n", childNodesCount, fileName);
```

Comment:
```suggestion
		ShowStatus("Loading '" CL_WHITE "%" PRIdPTR CL_RESET "' entries in '" CL_WHITE "%s" CL_RESET "'" CL_CLL "\r", childNodesCount, fileName);
```

Can it change from \n to \r  to improve readability?

Reply:
How would it improve readability?
This is what it looks like:
```
[02:12:44][Status]: Loading '5' entries in 'npc/re/merchants/barters/Extended_Ammunition.yml'
[02:12:44][Status]: Done reading '5' entries in 'npc/re/merchants/barters/Extended_Ammunition.yml'
[02:12:44][Status]: Loading '1' entries in 'npc/re/merchants/barters/Gemstone_Bagger.yml'
[02:12:44][Status]: Done reading '1' entries in 'npc/re/merchants/barters/Gemstone_Bagger.yml'
```